src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -72,6 +84,12 @@ configurationRegistry.registerConfiguration({
 			},
 			'default': []
 		},
+		'terminal.integrated.rightClickAction.windows': {
```

Comment:
I'm thinking instead of having 3 settings for each platform and using strings, instead having a single setting for all platforms that just turns on rightClickCopy/Paste. Maybe `terminal.integrated.rightClickCopyPaste`? Then the default could be true when `platform.isWindows` (put the default in terminal.ts and import it into this file).